.github/workflows/k4-r1-compatibility-normalization.yml

```
@@ -0,0 +1,240 @@
+permissions:
+  contents: read
+
+jobs:
+  compatibility-normalization:
+    name: k4-r1-compatibility-normalization
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    env:
+      K4_R1_IMPLEMENTATION_BASELINE_SHA: "f64a5f1ddbe57970d214c2f7b042fa421a0562fa"
+      K4_R1_IMPLEMENTATION_BRANCH: "codex/k4-r1-compatibility-normalization"
+      UV_PROJECT_ENVIRONMENT: ${{ runner.temp }}/k4-r1-schema-venv
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)